Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -107,7 +104,6 @@
 {
   entities(
     scope: "API"
-    limit: $limit
```

Comment:
Same reason. I was suggested by my product team that we do not need this since we are explicitly providing the api ids:
`        
fields_list = Helper.construct_field_attribute_expression(self.optional_api_attributes)
        return Template(api_entities_query).substitute(
            filter_by_clause=filter_by_clause,
            fields_list=fields_list,
            limit=self.limit,
            starttime=Helper.start_datetime_to_string(starttime),
            endtime=Helper.end_datetime_to_string(endtime),
        )
`